mod-source-record-manager-server/src/main/java/org/folio/services/JobExecutionServiceImpl.java

```diff
@@ -157,6 +157,7 @@
             jobExecution.setUiStatus(JobExecution.UiStatus.fromValue(Status.valueOf(status.getStatus().name()).getUiStatus()));
             if (status.getStatus().equals(StatusDto.Status.ERROR)) {
               jobExecution.withErrorStatus(JobExecution.ErrorStatus.fromValue(status.getErrorStatus().name()))
+                .withProgress(jobExecution.getProgress().withTotal(0))
```

Comment:
Please update progress in place where errorStatus is set for JobExecution (check if it is FILE_PROCESSING_ERROR).